examples/mask.py

```diff
@@ -19,7 +19,7 @@ def maskFromSurface(surface, threshold = 127):
     if key:
         for y in range(surface.get_height()):
             for x in range(surface.get_width()):
-                if surface.get_at((x+0.1,y+0.1)) != key:
+                if surface.get_at((x,y)) != key:
```

Comment:
This is worth further investigation imho. Does `Surface.get_at work` with floats on SDL1 but not on SDL2?

Reply:
Oh. Interesting. I'm confused why floats were used there. Oh well. It will remain a mystery.